lib/search/searchSnippet.ts

```
@@ -0,0 +1,66 @@
+  }
+
+  // Match is deep in text, create snippet centered around it
+  const contextBefore = Math.floor((maxLength - matchLength) / 2);
+  const contextAfter = maxLength - matchLength - contextBefore;
+
+  let start = Math.max(0, firstMatchIndex - contextBefore);
+  let end = Math.min(text.length, firstMatchIndex + matchLength + contextAfter);
+
+
+  if (start > 0) {
+    const wordStart = text.lastIndexOf(' ', start);
```

ESLint

[error] Replace `'·'` with `"·"`

(prettier/prettier)